gecode/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_name=gecode
+pkg_distname=$pkg_name
+pkg_origin=core
+pkg_version=3.7.3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gecode')
+pkg_source=http://www.gecode.org/download/gecode-${pkg_version}.tar.gz
+pkg_shasum=e7cc8bcc18b49195fef0544061bdd2e484a1240923e4e85fa39e8d6bb492854c
+pkg_dirname=${pkg_distname}-${pkg_version}
+pkg_deps=(core/glibc core/gcc)
+pkg_build_deps=(core/make core/gcc core/coreutils core/diffutils core/perl)
+
```

Comment:
@tyler-ball What do you see in the "Installed Path".  You can find the installed path at the end of the build log, it will look something like this:

```
Installed Path: /hab/pkgs/core/libpcap/1.7.4/20160727210527
```

Without exporting an include and lib dir, other packages that build against this will have no way to find the header files or the shared libraries provided by gecode. It might not be `lib` and `include` exactly but what you are looking for are the directories in the install folder that have the `.h` files and the `.so` files.